components/config-provider/renderEmpty.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import Empty from '../empty';
+import { ConfigConsumer, ConfigConsumerProps } from './';
+import emptyImg from './empty.svg';
+
+const renderEmpty = (componentName?: string): React.ReactNode => (
+  <ConfigConsumer>
+    {({ getPrefixCls }: ConfigConsumerProps) => {
+      const prefix = getPrefixCls('empty');
+
+      switch (componentName) {
```

Comment:
Any specific reasons for passing `componentName` not just 'default' and 'small'?